typescript/sdk/src/ism/HyperlaneIsmFactory.ts

```diff
@@ -220,18 +224,55 @@ export class HyperlaneIsmFactory extends HyperlaneApp<ProxyFactoryFactories> {
     logger: Logger,
   ): Promise<IMultisigIsm> {
     const signer = this.multiProvider.getSigner(destination);
-    const multisigIsmFactory =
-      config.type === IsmType.MERKLE_ROOT_MULTISIG
-        ? this.getContracts(destination).staticMerkleRootMultisigIsmFactory
-        : this.getContracts(destination).staticMessageIdMultisigIsmFactory;
 
-    const address = await this.deployStaticAddressSet(
-      destination,
-      multisigIsmFactory,
-      config.validators,
-      logger,
-      config.threshold,
-    );
+    const deployStatic = (factory: StaticThresholdAddressSetFactory) =>
```

Comment:
will need to dupe this change to `EvmIsmModule` too https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/471002bd1a3c1d36e26bf7f54c5d0bed70bf1543/typescript/sdk/src/ism/EvmIsmModule.ts#L433

Reply:
brutal, I thought these were kinda using the same codepaths 😱